### PR TITLE
Plumb through out of date notification

### PR DESCRIPTION
**Pull request by chrisnojima**
to repro you can comment out this line
https://github.com/keybase/client/blob/master/go/libkb/api.go#L345

@keybase/react-hackers 

screen shot 2016-05-23 at 7 53 59 pm

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
:mag_right: :zap: These commits introduced some visual changes:
 * Changed: **Folders_TLF-Normal** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1464048204646-f7c81e9f5723-429465b422a8/Folders_TLF-Normal-f7c81e9f5723.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1464048204646-f7c81e9f5723-429465b422a8/Folders_TLF-Normal-429465b422a8.png)  
   Folders_TLF-Normal comparison

File: shared/native/notification-listeners.desktop.js [hunk 1]
```diff
@@ -32,6 +33,10 @@ export default function (dispatch: Dispatch, getState: () => Object, notify: any
       dispatch(bootstrap())
       response.result()
     },
+    'keybase.1.NotifySession.clientOutOfDate': ({upgradeTo, upgradeURI, upgradeMsg}) => {
+      const body = upgradeMsg || `Please update to ${upgradeTo} by going to ${upgradeURI}`
+      NotifyPopup('Client out of date!', {body})
```
Comment (chromakode) [inline shared/native/notification-listeners.desktop.js]: Can you use the `notify` callback here? It looks like it's just equal to `NotifyPopup`: https://github.com/keybase/client/blob/429465b422a890105fb8b2271802eabea7d997d8/shared/actions/notifications.js#L31

Comment (chrisnojima) [inline shared/native/notification-listeners.desktop.js]: updated. i really dislike this notify param. why even bother doing this? also the fact that its typed as 'any' means its hard to understand. i'm changing it so its like the rest but i think this is an anti-pattern

**Comment by keybase-ci-visdiff** [comment]
:mag_right: :zap: These commits introduced some visual changes:
 * Changed: **Folders_TLF-Normal** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1464098391525-f7c81e9f5723-c53ab50855b7/Folders_TLF-Normal-f7c81e9f5723.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1464098391525-f7c81e9f5723-c53ab50855b7/Folders_TLF-Normal-c53ab50855b7.png)  
   Folders_TLF-Normal comparison

**Comment by chromakode** [comment]
:+1:

**Comment by MarcoPolo** [comment]
👍

**Pull request merged**